tools/lang-handler-aliases.js

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env node
```

Comment:
Note to self: port of https://github.com/google/code-prettify/blob/master/tools/lang-handler-aliases.sh